models/validators/UDFValidator.cfc

```diff
@@ -34,9 +34,14 @@ component extends="BaseValidator" accessors="true" singleton {
 	){
 		var errorMetadata = {};
 
+        // return true if no data to check, type needs a data element to be checked.
+		if ( isNull( arguments.targetValue ) || isNullOrEmpty( arguments.targetValue ) ) {
+			return true;
+		}
+
 		// Validate against the UDF/closure
 		var passed = arguments.validationData(
-			isNull( arguments.targetValue ) ? javacast( "null", "" ) : arguments.targetValue,
+			arguments.targetValue,
```

Comment:
I am thinking that if the value is actually `null`, this will fail, did you try it?